src/style.css

```diff
@@ -0,0 +1,100 @@
+@import "./styles/normalize.css";
```

Comment:
all block styles should be in separate file. 11 cl [FILE STRUCTURE] - You should create separate file for each BEM block